templates/cisco_wlc_ssh_show_band-select.textfsm

```diff
@@ -0,0 +1,19 @@
+Value BAND_SELECT_PROBE_RESPONSE (.+?)
+Value BAND_SELECT_CYCLE_COUNT (\d+ \w+)
+Value BAND_SELECT_CYCLE_THRESH (\d+ \w+)
+Value BAND_SELECT_AGE_SUPPRESS (\d+ \w+)
+Value BAND_SELECT_AGE_DUAL_BAND (\d+ \w+)
+Value BAND_SELECT_CLIENT_RSSI (-\d+ dBm)
+Value BAND_SELECT_CLIENT_MID_RSSI (-\d+ dBm)
+
+
+Start
+  ^Band\s+Select\s+Probe\s+Response\.*\s${BAND_SELECT_PROBE_RESPONSE}s*$$
```

Comment:
There is still some static spacing on all the rule lines `\s`.

Also, can you change the value regex's to have `\s+` as well?